cli/cmd/resources/scheduler.go

```diff
@@ -31,13 +37,13 @@ func NewSchedulerServiceAccount(ns string) *corev1.ServiceAccount {
 			APIVersion: "v1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "odigos-scheduler",
+			Name:      SchedulerServiceAccountName,
 			Namespace: ns,
 		},
 	}
 }
 
-func NewSchedulerRoleBinding(ns string) *rbacv1.RoleBinding {
+func NewSchedulerLeaderElectionRoleBinding(ns string) *rbacv1.RoleBinding {
 	return &rbacv1.RoleBinding{
```

Comment:
could we just embed the role and rolebinding yamls and marshal them to the API types? Then we can keep them in one spot that's a little more readable

Reply:
Yes, they are currently duplicated between helm and cli. we sync them manually but need to find a better way to have them just once

Reply:
Opened https://github.com/odigos-io/odigos/issues/2047 to track (gh issue since it's not high priority)